tests/unit/server/bundle_manager/__init__.py

```diff
@@ -0,0 +1,120 @@
+import unittest
+from mock import Mock
+
+from codalab.server.bundle_manager import BundleManager
+from codalab.worker.bundle_state import BundleCheckinState
+from codalab.lib.codalab_manager import CodaLabManager
+from codalab.lib.spec_util import generate_uuid
+
+
+BASE_METADATA = {
+    "docker_image": "sckoo/bird-brain@sha256:5076a236533caf8bea3410dcfaa10ef2dab506a3505cd33bce5190951d99af84",
+    "time": 1830.8628242,
+    "started": 1495784349,
+    "request_network": False,
+    "request_cpus": 0,
+    "request_memory": "0",
+    "request_time": "0",
+    "request_priority": 0,
+    "description": "",
+    "request_queue": "",
+    "name": "run-python",
+    "exitcode": 137,
+    "data_size": 601111,
+    "created": 1495784349,
+    "allow_failed_dependencies": False,
+    "actions": ["kill"],
+    "request_docker_image": "sckoo/bird-brain:v3",
+    "memory_max": 0,
+    "tags": [],
+    "run_status": "Finished",
+    "last_updated": 1495786180,
+    "failure_message": "Kill requested",
+    "request_disk": "",
+    "request_gpus": 0,
+    "remote": "vm-clws-prod-worker-3",
+    "exclude_patterns": [],
+}
+
+BASE_METADATA_MAKE_BUNDLE = {
+    "description": "",
+    "name": "run-python",
+    "created": 1495784349,
+    "failure_message": "Kill requested",
```

Comment:
Why is there a failure message here?  Seems odd to be the default for all these bundles